### PR TITLE
Task #49344 [Admin] Edit Room View

### PR DESCRIPTION
**Pull request by anhlh-3049**
## Related Tickets
- https://edu-redmine.sun-asterisk.vn/issues/49344

## WHAT (optional)
- Edit Room View

## Evidence (Screenshot or Video)
Screenshot_13

### CODE REVIEW

**Comment by anhlh-3049** [comment]
ready

File: app/controllers/admin/room_views_controller.rb [hunk 1]
```diff
@@ -7,17 +9,42 @@ def new
...
+
+  def destroy
+    @room_view.destroy
+    flash[:success] = t ".flash_sucsses"
+    redirect_to admin_room_views_path
+  end
+
   private
+  def find_room_view
+    @room_view = RoomView.find_by(id: params[:id])
```
Comment (ngocvt-0484) [inline app/controllers/admin/room_views_controller.rb]: sao code giống nhau vậy e

File: app/controllers/admin/rooms_controller.rb [hunk 2]
```diff
@@ -25,16 +26,24 @@ def edit; end
...
+  def destroy
+    @room.destroy
+    flash[:success] = "Delete success"
+    redirect_to admin_rooms_path
+  end
+
   private
   def find_room
     @room = Room.find_by(id: params[:id])
+    flash[:danger] = t ".flash_error"
```
Comment (ngocvt-0484) [inline app/controllers/admin/rooms_controller.rb]: pull giống nhau quá

File: app/views/admin/rooms/index.html.erb [hunk 3]
```diff
@@ -32,6 +33,8 @@
               <td>
                 <div class="sparkbar"><%= room.description %></div>
               </td>
+              <td><%= link_to "Delete", [:admin, room], method: :delete,
+                                          data: { confirm: "You want delete?" } %></td>
```
Comment (ngocvt-0484) [inline app/views/admin/rooms/index.html.erb]: i18n

File: app/views/admin/room_views/index.html.erb [hunk 4]
```diff
@@ -28,4 +29,5 @@
   <div class="card-footer clearfix">
     <%= link_to t(".create_btn"), new_admin_room_view_path, class: "btn btn-sm btn-info float-left" %>
   </div>
+<%== pagy_bootstrap_nav(@pagy) %>
```
Comment (ngocvt-0484) [inline app/views/admin/room_views/index.html.erb]: indent

**Comment by anhlh-3049** [comment]
ready

**Pull request merged**